migrations/1613491062486-replaceFileLinksHomework.js

```diff
@@ -0,0 +1,136 @@
+const mongoose = require('mongoose');
+// eslint-disable-next-line no-unused-vars
+const { alert, error } = require('../src/logger');
```

Comment:
`alert` is even more severe than an error.
`warning` or `notice` seems to be more appropriate in all places it is used here.